Source/town.cpp

```diff
@@ -1280,7 +1280,7 @@ void SetTownMicros()
 
 	if (zoomflag) {
 		scr_pix_width = 640;

```

Comment:
Let's also use a const for this, either SCREEN_WIDTH or define `VIEWPORT_WIDTH = SCREEN_WIDTH`

Reply:
That is still WIP. I want to tackle them one at a time. It's confusing enough as it is.

Reply:
Ok, in that case lets let this PR cook a bit longer before we merge it :)

Reply:
I would prefer to merge it in parts. It's confusing enough to split changes between both repos as it is.